SplunkRumWorkspace/SplunkRum/SplunkRum/NetworkInstrumentation.swift

```diff
@@ -68,9 +70,13 @@ func endHttpSpan(span: Span?, task: URLSessionTask) {
         span?.setAttribute(key: "net.host.connection.type", value: hostConnectionType!)
     }
     span!.end()
+    print("http span ENDED....")
 }
 
 func startHttpSpan(request: URLRequest?) -> Span? {
+    print("Request")
```

Comment:
Can we get rid of the printouts? Seem to be unrelated to this PR anyway

Reply:
Done.